src/main/java/org/spectrumauctions/sats/core/model/bvm/BMWorldSetup.java

```diff
@@ -78,29 +74,29 @@ public boolean removeBand(String nameOfBand) {
         }
 
         /**
-         * See {@link BMWorldSetup#bands()} for the purpose of this parameter
+         * @see BMWorldSetup#bands() for the purpose of this parameter
```

Comment:
Are you sure this is what you want to do?  @see creates a separate javadoc line. If used within text (as done here), its typical to use @link. If using  @see, you may want to remove the "for the purpose of this parameter". 

I'm not 100% sure about that, though :-D

Reply:
No, that's definitely not the purpose, thanks for pointing it out. I'm using the link again.